.gitignore

```
@@ -127,6 +127,9 @@ editor/src/components/editor/npm-dependency/utopia-api-typings.ts
 .github-test-projects
 
 # Database.
-server/pglog.txt
-server/.pglock
-server/utopia-db
+pglog.txt
+.pglock
+utopia-db
```

Looking forward to the upcoming PRs with the previous files now suddenly included :grinning:

---

Muahahaha.

---

are you trying to suggest we should keep the old one sticking around?
```suggestion
pglog.txt
.pglock
server/utopia-db
utopia-db
```